### ISSUE TITLE
Accessibility issue - All page content should be contained by landmarks

### ISSUE THREAD

**Issue by jaimeiniesta** [buunguyen/topbar#12]
Hi! Thanks for building this, it works great :)

There's just a little detail, I checked for accessibility issues and Axe complains about this:

![Captura de pantalla 2021-06-17 a las 11 35 32](https://user-images.githubusercontent.com/2629/122371410-23d67780-cf60-11eb-83c6-14a26afa50e9.png)

In short, it's because the `<canvas>` used is not contained inside a landmark. I think that a nice way to fix this would be specifying the element that should be the parent of that `<canvas>`, defaulting to `<body>` as it does now. 

For example, I could configure Topbar to use the `<header>` element this way:

```js
topbar.config({
    container  : 'header'
}’
```

where `container` could be the identifier of any DOM element.

What do you think?

**Comment by buunguyen** [comment]
I would appreciate a pull request for this. Thanks.

**Comment by andresgutgon** [comment]
Did this here:
https://github.com/buunguyen/topbar/pull/20

**Comment by James-E-A** [comment]
> All page content must be contained by landmarks
>
> Rule ID: `region`  
> Ruleset: axe-core 4.1  
> User Impact: Moderate  
> Guidelines: Deque Best Practice

Considering that this element *isn't* "page content", and is going to be essentially unusable for most assistive technologies (braille, screen reader, etc.), wouldn't it be much more appropriate to just mark it as [presentation](https://www.w3.org/WAI/ARIA/apg/practices/hiding-semantics/) than to try to arbitrarily assign it a landmark?

> common uses of role `presentation` \[include\] Hiding a decorative image…

-----

```diff
--- a/topbar.js
+++ b/topbar.js
@@ -54,4 +54,5 @@
     createCanvas = function () {
       canvas = document.createElement("canvas");
+      canvas.role = "presentation";
       var style = canvas.style;
```

**Comment by Flo0807** [comment]
> > All page content must be contained by landmarks
> > Rule ID: `region`
> > Ruleset: axe-core 4.1
> > User Impact: Moderate
> > Guidelines: Deque Best Practice
> 
> Considering that this element _isn't_ "page content", and is going to be essentially unusable for most assistive technologies (braille, screen reader, etc.), wouldn't it be much more appropriate to just mark it as [presentation](https://www.w3.org/WAI/ARIA/apg/practices/hiding-semantics/) than to try to arbitrarily assign it a landmark?
> 
> > common uses of role `presentation` [include] Hiding a decorative image…
> 
> --- a/topbar.js
> +++ b/topbar.js
> @@ -54,4 +54,5 @@
>      createCanvas = function () {
>        canvas = document.createElement("canvas");
> +      canvas.role = "presentation";
>        var style = canvas.style;

I've submitted a PR and added you as the co-author: #23 

@buunguyen
